### PR TITLE
Start rclone containers

### PR DESCRIPTION
**Pull request by vtemian**
Starts Wordpress with `UPLOADS_FTP_HOST` and `UPLOADS_HTTP_PROXY` that point to newly added rclone containers.

#### Tests
- use `wordpress-chart` to create a new site
- set `bucket`, `project_id` and `google_credentials`
- upload a photo
- upload a pdf
- those files should be served

### CODE REVIEW

File: pkg/internal/wordpress/pod_template_test.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+				GitDir: &wordpressv1alpha1.GitVolumeSource{},
+			}
+			w.Spec.MediaVolumeSpec = &wordpressv1alpha1.MediaVolumeSpec{
+				GCSVolumeSource: &wordpressv1alpha1.GCSVolumeSource{Env: []corev1.EnvVar{}},
+			}
+
+			spec := podSpec()
+
+			initContainers := spec.Spec.InitContainers
+			Expect(len(initContainers)).To(Equal(2))
```
Comment (calind) [inline pkg/internal/wordpress/pod_template_test.go]:
```suggestion
			Expect(initContainers).To(HaveLen(2))
```

File: pkg/internal/wordpress/pod_template_test.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+			Expect(initContainers[0].Name).To(Equal("rclone-init-ftp"))
+			Expect(initContainers[0].Image).To(Equal(rcloneImage))
+
+			Expect(initContainers[1].Name).To(Equal("git"))
+			Expect(initContainers[1].Image).To(Equal(gitCloneImage))
+
+			containers := spec.Spec.Containers
+			Expect(len(containers)).To(Equal(3))
```
Comment (calind) [inline pkg/internal/wordpress/pod_template_test.go]:
```suggestion
			Expect(containers).To(HaveLen(3))
```

File: pkg/internal/wordpress/pod_template_test.go [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+	DescribeTable("Should generate just a git container, when no remote stream is configure",
+		func(f func() (func() corev1.PodTemplateSpec, *Wordpress)) {
+			podSpec, w := f()
+
+			w.Spec.CodeVolumeSpec = &wordpressv1alpha1.CodeVolumeSpec{
+				GitDir: &wordpressv1alpha1.GitVolumeSource{},
+			}
+			containers := podSpec().Spec.InitContainers
+
+			Expect(len(containers)).To(Equal(1))
```
[the remark this answers is not in the document]
Comment (vtemian) [reply pkg/internal/wordpress/pod_template_test.go]: Done.

File: pkg/internal/wordpress/pod_template_test.go (hunk 1 above)
Comment (vtemian) [reply pkg/internal/wordpress/pod_template_test.go]: Done.

File: pkg/internal/wordpress/pod_template_test.go (hunk 2 above)
Comment (vtemian) [reply pkg/internal/wordpress/pod_template_test.go]: Done.

File: pkg/internal/wordpress/pod_template_test.go [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+	DescribeTable("Shouldn't generate any init containers, if git and a remote stream is not configured",
+		func(f func() corev1.PodTemplateSpec) {
+			Expect(f().Spec.InitContainers).To(Equal([]corev1.Container{}))
+		},
+		Entry("for web pod", func() corev1.PodTemplateSpec { return wp.WebPodTemplateSpec() }),
+		Entry("for job pod", func() corev1.PodTemplateSpec { return wp.JobPodTemplateSpec() }),
+	)
+
+	DescribeTable("Should generate just a git container, when no remote stream is configure",
```
[the remark this answers is not in the document]
Comment (vtemian) [reply pkg/internal/wordpress/pod_template_test.go]: Done.

File: pkg/internal/wordpress/pod_template_test.go [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+			Spec: wordpressv1alpha1.WordpressSpec{
+				Domains: []wordpressv1alpha1.Domain{"test.com"},
+			},
+		})
+		wp.SetDefaults()
+	})
+
+	DescribeTable("Shouldn't generate any init containers, if git and a remote stream is not configured",
+		func(f func() corev1.PodTemplateSpec) {
+			Expect(f().Spec.InitContainers).To(Equal([]corev1.Container{}))
```
[the remark this answers is not in the document]
Comment (vtemian) [reply pkg/internal/wordpress/pod_template_test.go]: Done.

File: pkg/internal/wordpress/pod_template.go [hunk 6]
```diff
@@ -284,6 +264,90 @@ func (wp *Wordpress) gitCloneContainer() corev1.Container {
 	}
 }
 
+func (wp *Wordpress) rcloneContainer(name string, args []string) corev1.Container {
+	var env []corev1.EnvVar
+	if wp.Spec.MediaVolumeSpec.S3VolumeSource != nil {
```
[the remark this answers is not in the document]
Comment (vtemian) [reply pkg/internal/wordpress/pod_template.go]: Done.

**Pull request merged**